cmd/influxdb-writer/main.go

```diff
@@ -80,8 +85,13 @@ func main() {
 	}
 	defer client.Close()
 
-	timeout := time.Duration(cfg.BatchTimeout) * time.Second
-	repo, err := influxdb.New(client, cfg.DBName, cfg.BatchSize, timeout)
+	batchTimeout, batchSize, err := unmarshalInfluxDBSettings(cfg.BatchTimeout, cfg.BatchSize)
```

Comment:
No need for this function `unmarshalInfluxDBSettings()` IMHO. I think it hurts readability.

I would rather go with:
```batchTimeout, err = strconv.Atoi(cfgBatchTimeout)
	if err != nil {
		// log error
                // os.Exit(1)
	}
 	batchSize, err = strconv.Atoi(cfgBatchSize)
	if err != nil {
		// log error
                // os.Exit(1)
	}
```
directly here.

@anovakovic01 @dusanb94 opinions?

Reply:
@drasko I agree. @mfinley3 However, if you want to keep this method, I suggest you rename to something like `loadInfluxDBConfig`. I also suggest not to return an error but to simply log and kill in case of wrong env variable values.

Reply:
I agree as well, I think the function hurts readability. I will remove the function and get this looking better

Reply:
I decided that it flowed better without calling a separate function, so went with keeping these in main().